### ISSUE TITLE
save language tags in lower case to ensure consistency

### ISSUE THREAD

**Issue by samuelstroschein** [opral/inlang-message-sdk#115]
## Problem 

Language tags are inconsistent across inlang apps because they are case sensitive. See [https://github.com/opral/inlang-fink/issues/40](https://github.com/opral/inlang-fink/issues/40). 

## Proopsal

Store language tags always in lower case in an inlang project. 

* According to BCP47, casing is irrelevant
* Human readability of casing might be increased but the cost of enforcing case insensitivity across inlang apps is too high

**Comment by samuelstroschein** [comment]
@loris.sigrist do you think this is a good idea?

**Comment by LorisSigrist** [comment]
Yes, Paraglide already noramlizes the case for all comparisons. It would be good to have some consistency

**Comment by samuelstroschein** [comment]
cancelling. this is the job of a validation rule. if we implicitly save locales in lower case, i am sure that users will be surprised.
